src/components/switch/index.tsx

```diff
@@ -153,9 +155,9 @@ class Switch extends BaseComponent {
   }
 
   render() {
-    const {...others} = this.getThemeProps();
-
+    const {...others} = this.props;
     return (
+      // @ts-ignore
```

Comment:
Did you get an error here (I don't), not sure why (because I think we got those for the accessibility props).
Maybe the new change in typescript solved it, can you try to clean-install and check?

Reply:
I clean-installed and still get an error here..

Reply:
Yes, it seems I had a problem on my end (restarted VS Code and I have an error).